File: Models/UserRegisterModel.cs
using Bug_Tracker.Enums;

namespace Bug_Tracker.Models;

public class UserRegisterModel
{
    public string Email { get; set; } // email
    public string Password { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string Place { get; set; }
}

File: ClientApp/bugtracker/src/app/models/UserRegisterModel.ts
export class UserRegisterModel {
  firstName : string | null;
  lastName : string | null;
  place : string | null;
  email : string | null;
  password : string | null;

  constructor(firstName: string | null, lastName: string | null, place: string | null, email: string | null, password: string | null) {
    this.firstName = firstName;
    this.lastName = lastName;
    this.place = place;
    this.email = email;
    this.password = password;
  }
}
